augly/image/transforms.py

```diff
@@ -212,7 +212,6 @@ def apply_transform(
         """
         return F.brightness(image, factor=self.factor, metadata=metadata)
 
-
```

Comment:
add back in newline; there should be two newlines between classes

Reply:
Done!!